Makefile

```
@@ -180,8 +181,10 @@ build-push-e2e-catalog: ## Build the testdata catalog used for e2e tests and pus
 .PHONY: test-e2e
 test-e2e: KIND_CLUSTER_NAME := operator-controller-e2e
 test-e2e: KUSTOMIZE_BUILD_DIR := config/overlays/e2e
+test-e2e: GOOS=linux
+test-e2e: GOARCH=amd64
```

Use `export` here.

---

Would that not force these values across the entire makefile? Is that what we want?

---

Oh nevermind, I think I see what you mean. Like this?
```suggestion
test-e2e: export GOOS=linux
test-e2e: export GOARCH=amd64
```

---

Updated.